requirements.txt

```diff
@@ -1,2 +1,4 @@
 pydap==3.2.2
-requests==2.25.1
\ No newline at end of file
+requests==2.25.1
```

Comment:
Does anything actually use `requests`? I don't see it show up in the code...

```bash
(venv) james@basalt:~/code/git/pydap-extras$ grep -r requests *
pydap_extras/handlers/sql.py:        for incoming requests.
Binary file pydap_extras/handlers/__pycache__/sql.cpython-36.pyc matches
Binary file pydap_extras/responses/__pycache__/xlsx.cpython-36.pyc matches
pydap_extras.egg-info/requires.txt:requests
requirements.txt:requests==2.25.1
setup.py:    install_requires=["pydap", "requests"],
```

Reply:
It seems to be a dependency of pydap, but isn't installed with it
```bash
venv/lib/python3.8/site-packages/pydap/net.py:4: in <module>
    import requests
E   ModuleNotFoundError: No module named 'requests'

```